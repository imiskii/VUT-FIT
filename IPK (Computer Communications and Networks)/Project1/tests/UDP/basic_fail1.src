(/ 8 0)
bad msg
(+ 5 7)