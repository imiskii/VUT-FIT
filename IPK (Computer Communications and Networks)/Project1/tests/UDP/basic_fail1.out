ERR
ERR
OK:12

